R/prep_data.R

```diff
@@ -60,7 +60,9 @@
 #' @param add_levels Logical. If TRUE (defaults), "other" and "hcai_missing"
 #'   will be added to all nominal columns. This is protective in deployment: new
 #'   levels found in deployment will become "other" and missingness in
-#'   deployment can become "hcai_missing".
+#'   deployment can become "hcai_missing". If FALSE, these levels may be added
```

Comment:
I don't like the user experience of "May" be added. Can I get more information as to why this appears non-deterministic from a user perspective?

Reply:
Add to docs; to answer your question: if the imputation method of categorical variables is to create a new level, then missing will become "hcai_missing".